beta/nncf/tensorflow/sparsity/base_algorithm.py

```diff
@@ -62,5 +63,7 @@ def statistics(self, quickly_collected_only: bool = False):
         stats.update(convert_raw_to_printable(raw_sparsity_statistics, 'sparsity', header))
         return stats
 
+
+    @abstractmethod
     def raw_statistics(self):
-        raise NotImplementedError
+        ...
```

Comment:
Should use `pass`

Reply:
I followed our [styleguide](https://github.com/openvinotoolkit/nncf/blob/develop/docs/styleguide/PyGuide.md#29-abstract-classes), should we fix that place in styleguide?

Reply:
It should be fixed, yes. The `...` is actually a valid identifier in Python of the `Ellipsis` type, so your code works, but this is not proper style, really.

Reply:
Ok. Please watch https://github.com/openvinotoolkit/nncf/pull/680